[2023-03-22 08:32:25,264] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: load_to_DWlayer.sukses_Proses_Load_ToDW manual__2023-03-22T08:29:47.330252+00:00 [queued]>
[2023-03-22 08:32:25,284] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: load_to_DWlayer.sukses_Proses_Load_ToDW manual__2023-03-22T08:29:47.330252+00:00 [queued]>
[2023-03-22 08:32:25,285] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-03-22 08:32:25,285] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2023-03-22 08:32:25,286] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-03-22 08:32:25,299] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): sukses_Proses_Load_ToDW> on 2023-03-22 08:29:47.330252+00:00
[2023-03-22 08:32:25,303] {standard_task_runner.py:52} INFO - Started process 707 to run task
[2023-03-22 08:32:25,306] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'load_to_DWlayer', 'sukses_Proses_Load_ToDW', 'manual__2023-03-22T08:29:47.330252+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/etl_warehouse.py', '--cfg-path', '/tmp/tmpzjm3nso9', '--error-file', '/tmp/tmpdp98xafv']
[2023-03-22 08:32:25,307] {standard_task_runner.py:77} INFO - Job 16: Subtask sukses_Proses_Load_ToDW
[2023-03-22 08:32:25,347] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_DWlayer.sukses_Proses_Load_ToDW manual__2023-03-22T08:29:47.330252+00:00 [running]> on host 2bbf1c230888
[2023-03-22 08:32:25,393] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_DWlayer
AIRFLOW_CTX_TASK_ID=sukses_Proses_Load_ToDW
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T08:29:47.330252+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-22T08:29:47.330252+00:00
[2023-03-22 08:32:25,394] {logging_mixin.py:109} INFO - Etl Sukses Dilakukan
[2023-03-22 08:32:25,395] {python.py:152} INFO - Done. Returned value was: None
[2023-03-22 08:32:25,406] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=load_to_DWlayer, task_id=sukses_Proses_Load_ToDW, execution_date=20230322T082947, start_date=20230322T083225, end_date=20230322T083225
[2023-03-22 08:32:25,439] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-22 08:32:25,467] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
